compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -245,6 +245,42 @@ void addGPUVectorizationPassPipeline(OpPassManager &funcPassManager) {
   funcPassManager.addPass(createOptimizeTensorInsertExtractSlicesPass());
 }
 
+//===---------------------------------------------------------------------===//
+// Winograd Vectorize
+//===---------------------------------------------------------------------===//
+
+void addGPUWinogradVectorizePassPipeline(OpPassManager &funcPassManager) {
+  tileAndDistributeToWorkgroup(funcPassManager);
+
+  funcPassManager.addPass(createCanonicalizerPass());
+  funcPassManager.addPass(createWorkgroupSpecializationPass());
```

Comment:
Do you need the workgroup specialization pass? I am planning to deprecate it, so want to make sure we dont have spiralling uses of it.

Reply:
Sorry, I forgot about this comment. I just verified performance/correctness, and I don't think this is necessary (actually seems to slightly improve benchmarks :) I will get rid of it.